generator/src/lib.rs

```diff
@@ -418,8 +421,25 @@ impl ConstantExt for vkxml::ExtensionEnum {
     }
 }
 
+impl ConstantExt for vk_parse::Enum {
+    fn constant(&self, enum_name: &str) -> Constant {
+        Constant::from_vk_parse_enum_spec(&self.spec, Some(enum_name), -1)
```

Comment:
Why is there a `-1` here?

Reply:
That's the default for `extension_number: i64,`, it is apparently used by things like `<enum extends="VkStructureType" extnumber="157" offset="0"          name="VK_STRUCTURE_TYPE_SAMPLER_YCBCR_CONVERSION_CREATE_INFO"/>`, where the actual value turns into `1_000_000_000 + (extnumber-1) * 1000 + offset`.

Good catch, this should be turned into an `Option<>` and unwrapped so that we won't get hidden invalid values!